intel/yaml/intel-istio-cryptomb.yaml

```diff
@@ -42,6 +42,7 @@ spec:
             privateKeyProvider:
               cryptomb:
                 pollDelay: 10ms
+                fallback: true
```

Comment:
Besides CryptoMB, can you also modify the qat configuration?

Reply:
Done